guides/release/getting-started/quick-start.md

```diff
@@ -260,6 +258,8 @@ export default class PeopleList extends Component {
 }
 ```
 
+The `@action` is something called a decorator, and it lets your Ember app know that the `showPerson` method should be available for the template to use.
```

Comment:
```suggestion
The `@action` is something called a decorator, and it lets your Ember app know that the `showPerson` method should be invocable by name in the component's template via `{{action "showPerson"}}`
```
Suggested rewording to avoid implying that we'd be unable to use find this method via `{{this.showPerson}}` even without the `@action` decorator